internal/graph/check.go

```diff
@@ -150,6 +150,10 @@ func (cc *ConcurrentChecker) checkSetOperation(ctx context.Context, req Validate
 			requests = append(requests, cc.checkUsersetRewrite(ctx, req, child.UsersetRewrite))
 		case *core.SetOperation_Child_TupleToUserset:
 			requests = append(requests, cc.checkTupleToUserset(ctx, req, child.TupleToUserset))
+		case *core.SetOperation_Child_XNil:
+			requests = append(requests, notMember())
+		default:
+			return checkError(errors.New("unknown set operation child in check"))
```

Comment:
Can you get the operation name in the error somehow? All errors should have some data that helps debug it.

Reply:
Yeah, I wasn't sure what to print out before, but `%T` seems to provide sufficient info without being too verbose